### PR TITLE
Add preview API key

### PR DESCRIPTION
**Pull request by winklertomas**
### Motivation

Possibility to get unpublished content

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

Try to get unpublished content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ This optional section allows you to create your own copy of the project in Konte
 | :--------------------------------: | :------: | :--------------------------------------------------------------------------------------- |
 |    REACT_APP_KONTENT_PROJECT_ID    |    NO    | Project identification                                                                   |
 | REACT_APP_KONTENT_GRAPHQL_ENDPOINT |    NO    | Kontent GraphQL endpoint                                                                 |
+| REACT_APP_KONTENT_PREVIEW_API_KEY  |    NO    | Preview API key to retrieve unpublished content                                          |
```
Comment (Simply007) [inline README.md:81]:
```suggestion
| REACT_APP_KONTENT_PREVIEW_API_KEY  |    NO    | Preview API key to retrieve unpublished content. If set, the application is fetching unpublished content, if not published content is being fetched.                                      |
```

**Review by Simply007 — CHANGES_REQUESTED** [review]
Just the extended preview information on Readme.

**Comment by winklertomas** [comment]
Thanks for the review -> will be merged after the release

**Review by Simply007 — APPROVED** [review]
OK

**Pull request merged**